src/core/common_path.py

```diff
@@ -0,0 +1,6 @@
+from pathlib import Path
+
+
+class CommonPaths:
+    project_root = Path(__file__).resolve().parent
```

Comment:
@vasylboliuk please use `resolve` method at the end of your path definition to return a new path relevant to your os dir structure

Reply:
Done